AsyncDisplayKit/Details/ASImageContainerProtocolCategories.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ASImageContainerProtocolCategories.h
```

Comment:
Although this file is short and simple, it might be worth merging into one of the other files such as ASImageNode+Internal.  If we should keep it, I think we would find greater parallelism in the name +Categories